src/optimizer_interface.jl

```diff
@@ -70,7 +70,9 @@ end
 """
     optimize!(model::Model,
               optimizer_factory::Union{Nothing, OptimizerFactory}=nothing;
-              ignore_optimize_hook=(model.optimize_hook === nothing))
+              bridge_constraints::Bool=true,
+              ignore_optimize_hook=(model.optimize_hook === nothing),
+              kwargs...)
 
 Optimize the model. If `optimizer_factory` is not `nothing`, it first sets the
```

Comment:
Please document that `kwargs` are passed to the `optimize_hook`, and it's an error to provide them otherwise.